django/utils/feedgenerator.py

```diff
@@ -52,8 +52,8 @@ def rfc2822_date(date):
     if offset is None:
         return time_str + '-0000'
     else:
-        timezone = (offset.days * 24 * 60) + (offset.seconds // 60)
-        hour, minute = divmod(timezone, 60)
+        timezone_ = (offset.days * 24 * 60) + (offset.seconds // 60)
```

Comment:
This is probably a subjective comment, but it would be nice to avoid code churn. I would suggest to import `utc` directly (instead of importing the timezone module and using it like `timezone.utc`.)